Source/EmmyLua.spoon/init.lua

```diff
@@ -183,28 +188,71 @@ function M.create(jsonDocs, prefix)
     module.name = prefix .. module.name
     local fname = options.annotations .. "/" .. module.name .. ".lua"
     local fmtime = hs.fs.attributes(fname, "modification")
+
     if fmtime == nil or mtime > fmtime then
-      -- print("creating " .. fname)
+      M.logger.i("creating " .. fname)
       local fd = io.open(fname, "w+")
       io.output(fd)
       M.processModule(module)
       io.close(fd)
+    else
+      M.logger.i("skipping " .. fname)
     end
   end
 end
 
+function M.createWhenChanged(jsonDocs, prefix)
+  local mtime = hs.fs.attributes(jsonDocs, "modification")
+  local timestamp = options.timestamps[jsonDocs]
+
+  if(timestamp == nil or mtime > timestamp) then
```

Comment:
Maybe better use here `~=` ?
```suggestion
  if(timestamp == nil or mtime ~= timestamp) then
```

for example If I checkout different commits from git then I can also go back and forth in time?